src/client/stdlib/fileAttachment.js

```diff
@@ -28,9 +28,10 @@ async function dsv(file, delimiter, {array = false, typed = false} = {}) {
 }
 
 export class AbstractFile {
-  constructor(name, mimeType = "application/octet-stream") {
-    Object.defineProperty(this, "name", {value: `${name}`, enumerable: true});
+  constructor(name, mimeType, lastModified) {
     Object.defineProperty(this, "mimeType", {value: `${mimeType}`, enumerable: true});
+    Object.defineProperty(this, "name", {value: `${name}`, enumerable: true});
+    if (lastModified !== undefined) Object.defineProperty(this, "lastModified", {value: Number(lastModified), enumerable: true}); // prettier-ignore
```

Comment:
Fine to always include this property, even if undefined?

Reply:
We still need a test because we don't want to show NaN for an undefined value?